src/uct/ib/dc/base/dc_ep.h

```diff
@@ -101,11 +101,12 @@ static inline void uct_dc_iface_dci_sched_tx(uct_dc_iface_t *iface, uct_dc_ep_t
  */
 
 enum uct_dc_ep_state {
-    UCT_DC_EP_TX_OK,
-    UCT_DC_EP_TX_WAIT,
-    UCT_DC_EP_INVALID
+    UCT_DC_EP_TX_WAIT   = UCS_BIT(0),
```

Comment:
need to rename "state" to "flags", since now it's flags and also "global" is not really a state:
```
enum {
UCT_DC_EP_FLAG_VALID  
UCT_DC_EP_FLAG_TX_WAIT   
UCT_DC_EP_FLAG_GRH /  UCT_DC_EP_FLAG_GLOBAL_ADDR
}
```
pls also add comment to every flag